xml/containers_basics.xml

```diff
@@ -1,78 +1,202 @@
+    isolated containers.
+  </para>
+  <itemizedlist mark="bullet" spacing="normal">
+    <title>Advantages of Using Containers</title>
+    <listitem>
+      <para>
+        Containers make it possible to solate applications through self-contained units.
+      </para>
+    </listitem>
+    <listitem>
+      <para>
+        Containers provide nearly native performance, as containers manage
+        allocation of resources in real time.
+      </para>
+    </listitem>
+    <listitem>
+      <para>
+        It is possible to control network interfaces and apply resources inside
+        containers through <xref linkend="gloss-vt-lxc-cgroups"/>.
+      </para>
+    </listitem>
+  </itemizedlist>
+  <itemizedlist mark="bullet" spacing="normal">
+    <title>Limitations of Containers</title>
+    <listitem>
+      <para>
+        Containers run inside the host system's kernel, and they cannot use
+        different kernels or different kernel versions.
+      </para>
+    </listitem>
+    <listitem>
+      <para>
+        Only Linux-based applications can be containerized.
+      </para>
+    </listitem>
+    <listitem>
+      <para>
+        Containers are not secure, and the overall security depends on the host
+        system. Containerized applications can be secured through &aa; or &selnx;
+        profile.
+      </para>
+    </listitem>
+  </itemizedlist>
+  <sect1 xml:id="sec-concepts-workflow">
+    <title>Key Concepts and Brief Introduction to &docker;</title>
+    <para>
+    &deng; is a well-established and mature container technology. While <xref
+    linkend="cha-docker-overview" /> provides an in-depth information on &deng;,
+    this chapter offers a quick introduction to key concepts and a basic
+    procedure of creating a container image an using it to run a container.
+    </para>
+    <para>
+      The basic &docker; workflow is as follows:
+    </para>
+    <para>
+      Registry -> Base image -> Dockerfile -> Custom image -> Container
+      <!--TODO replace with a figure-->
+    </para>
+
+    <para>
+      Running a container on a local machine or cloud service, usually involves the following steps:
+    </para>
+    <orderedlist>
+      <listitem>
+        <para>
+          Fetch a base image by pulling it from a registry to your local machine
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Create a Dockerfile and use it to build a custom image on top of the base image
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Use the created image to run one or multiple containers
+        </para>
+      </listitem>
+    </orderedlist>
+    <para>
+      To run a container, you need an image. An image includes everything needed to run a container: the code or binaries, runtimes, dependencies, and any other required components. For example, the SLE base image contains a slimmed-down version of the SLE distribution.
```

Comment:
```suggestion
      To run a container, you need an image. An image includes all the dependencies needed to run the application. For example, the SLE base image contains a slimmed-down version of the SLE distribution.
```

The SLE base image isn't really a "slimmed-down version", it's just a minimial package selection.